### PR TITLE
Asc 329/add molecule into cart

### PR DESCRIPTION
**Pull request by ShobhitExcelra**
Fixed Conflicts

### CODE REVIEW

File: apps/web/.eslintrc.js
Comment (himanshubexcelra) [inline apps/web/.eslintrc.js:1]: Why do you need to remove this file?

File: apps/api/.eslintrc.js
Comment (himanshubexcelra) [inline apps/api/.eslintrc.js:1]: Why this file needs to remove?

File: apps/web/.eslintrc.js
Comment (ShobhitExcelra) [reply apps/web/.eslintrc.js:1]: It's bad. I added again comparing with main branch.

File: apps/api/.eslintrc.js
Comment (ShobhitExcelra) [reply apps/api/.eslintrc.js:1]: It's my bad. I added again comparing with main branch.

**Comment by himanshubexcelra** [comment]
image

File: apps/web/components/Libraries/CartDetails.tsx [hunk 1]
```diff
@@ -93,7 +75,8 @@ export default function CartDetails({
             molecularWeight: item.molecular_weight,
             moleculeName: item.moleculeName,
             library_id: item.library_id,
-            project_id: item.project_id
+            project_id: item.project_id,
+            userId: userId
```
Comment (himanshubexcelra) [inline apps/web/components/Libraries/CartDetails.tsx:79]: Can we replace this userId?
with proper name like created_by
It's creating confustion

**Pull request merged**